tapdb/universe_test.go

```diff
@@ -562,13 +580,14 @@ func TestUniverseLeafQuery(t *testing.T) {
 			// key, so they must also share the same group witness.
 			switch {
 			case sharedWitness == nil:
-				sharedWitness =
-					leaf.GroupKey.Witness
+				sharedWitness = leaf.GroupKey.Witness
 			default:
-				leaf.GroupKey.Witness =
-					sharedWitness
-				leaf.Proof.Asset.GroupKey.Witness =
-					sharedWitness
+				leaf.GroupKey.Witness = sharedWitness
+
+				//nolint:lll
+				//leaf.Proof.Asset.GroupKey.Witness = sharedWitness
+				leaf.Asset.GroupKey.Witness = sharedWitness
+				// TODO(roasbeef): circle back
```

Comment:
What's the circle here, adding the group key to the proof? Confused why it doesn't fail as-is.

Reply:
Was a stray comment to remind myself to fix it after a rebase and tests started failing. The line above was for the old API where we stored the proof directly. The actual line just can access the asset in the leaf.